squarelet/users/views.py

```diff
@@ -155,6 +161,7 @@ class UserOnboardingView(TemplateView):
     template_name = "account/onboarding/base.html"  # Base template with includes
 
     def get_onboarding_step(self, request):
+        # pylint: disable=too-many-locals, too-many-return-statements
```

Comment:
This function is quite long, probably good to refactor it.  Maybe the logic for each step its own function?

Reply:
Yep, refactoring the onboarding logic is definitely the plan! I wanted to wait until we had all the steps in before figuring out the best way to refactor. I will open a new ticket for refactoring and testing the `UserOnboardingView`.

Reply:
https://github.com/MuckRock/squarelet/issues/323